### DISCUSSION TITLE
Welcome to greatpug-public Discussions!

### DISCUSSION THREAD

**Discussion by owlboy** [owlboy/greatpug-public#42] [General]
## 👋 Welcome!
  I've turned on Discussions to provide a place to connect with other members of The Great Pug community. You can:
  * Ask questions you’re wondering about.
  * Share ideas that don't really fit as bug reports or specific feature requests.
  * Engage with other community members.

**Comment by VenceVR** [comment]
any chance you'll remove chairs from the pug until VRChat fixes the issue with chairs? or at least add a toggle for it? its kind of ruining the experience of the pug :(

**Reply by owlboy** [reply]
Stations are currently disabled while the Saint Patrick's decorations are up. And I will be keeping them off, or adding a work around for them until the station bug is fixed.

**Reply by owlboy** [reply]
Stations are still disabled today. And there should be a VRChat patch in the coming week(s) that addressed the station issue. There is currently an open beta that contains the fix.

**Reply by VenceVR** [reply]
thank you!!

**Comment by Rexrove** [comment]
Hi Owlboy, the bar taps in the Great Pug, are you using a custom prefab for the pouring into glasses, or was it from a booth or some other store stie? I'm looking for the same thing for my world but am failing to find anything that matches. Thanks
